src/3dgfx/3dgfx.c

```diff
@@ -12,6 +12,9 @@
 #include "demo.h"
 #include "util.h"
 
+#if defined(__SUNPRO_C)
```

Comment:
This includes util.h, and you're fixing util.h below, so no need to double-fix it

Reply:
thanks, didn't notice that